modules/blogging/src/Volo.Blogging.Domain/Volo/Blogging/Posts/PostCacheInvalidator.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Volo.Abp.Caching;
+using Volo.Abp.DependencyInjection;
+using Volo.Abp.Domain.Entities.Events;
+using Volo.Abp.EventBus;
+
+namespace Volo.Blogging.Posts
+{
+    public class PostCacheInvalidator : ILocalEventHandler<EntityChangedEventData<Post>>, ITransientDependency
+    {
+        protected IDistributedCache<List<PostCacheItem>> Cache { get; }
+
+        public PostCacheInvalidator(IDistributedCache<List<PostCacheItem>> cache)
+        {
+            Cache = cache;
+        }
+        
+        public virtual async Task HandleEventAsync(EntityChangedEventData<Post> eventData)
+        {

```

Comment:
This event may be triggered when read count of a blog post changes, so it clears the cache unnecessarily.